Assets/Scripts/Managers/SaveManager.cs

```diff
@@ -96,10 +96,10 @@ public void Load(string fileName)
 
                 // set player state
                 PlayerState state = player.GetComponent<PlayerState>();
-                state.SetHealth(save.playerHealth);
-                state.SetHydration(save.playerHydration);
-                state.SetSanity(save.playerSanity);
-                state.SetSaturation(save.playerSaturation);
+                state.Health = save.playerHealth;
```

Comment:
Have you tested this? Are Savegames still working as intended?

Reply:
I didn't test savegames..

Reply:
As you made changes in the SaveManager, you should test it before we commit the changes...

Reply:
I think we should add some unit tests for this 🤔

Reply:
@Kaes3kuch3n srsly? haha
@potatoTVnet if you look at what I changed, you can see that the only thing I did, is making properties out of the getters and setters.. testing is always good, but in this case it wasn't worth the effort for me

Reply:
I didn't use savegames for now, I don't even have the newest version :( it would be nice if you could test for me if you want it to be tested

Reply:
Tested, still works fine @potatoTVnet